geonode/thumbs/thumbnails.py

```diff
@@ -271,12 +271,17 @@ def _datasets_locations(
             ]
         )
         if compute_bbox:
-            # handle exceeding the area of use of the default thumb's CRS
-            if (
+            if instance.ll_bbox_polygon:
+                _bbox = instance.ll_bbox_polygon.extent
```

Comment:
Is possible to convert those lines into a function? Since looks like that is equal to the same above.